packages/core/src/server/service.test.ts

```diff
@@ -1270,9 +1270,9 @@ test("throws if limit is greater than 1000", async (context) => {
   `);
 
   expect(response.body.errors[0].message).toBe(
-    "Invalid query. Cannot take more than 1000 rows."
+    "Invalid query. Cannot take more than 1000 rows. Received: 1005 rows."
   );
-  expect(response.statusCode).toBe(500);
+  expect(response.statusCode).toBe(200);
```

Comment:
@0xOlias After some thinking, I updated the expected status code here from `500` to `200`.

Since the expected error in this test is due to incorrect/bad input, this is more of a bad request than a [server error](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/500), so it makes sense that Yoga is returning status code `200` for this request.

Reply:
Makes perfect sense. OOC, what does the GraphiQL interface look like if you run a request that exceeds the limit?

Reply:
I haven't tried with the same exact query in that test but I tried in `ethfs`'s playground  
![CleanShot 2023-10-03 at 15 22 00@2x](https://github.com/0xOlias/ponder/assets/23618431/c5dae290-1717-4c1b-82b3-7ad917e42fd5)

Then when > 1k

![CleanShot 2023-10-03 at 15 23 41@2x](https://github.com/0xOlias/ponder/assets/23618431/e1ed2b1c-d0b5-4611-9227-039acf1ea25d)